codelab-master/apps/playground/src/app/quiz/containers/introduction/introduction.component.html

```diff
@@ -0,0 +1,40 @@
+<ng-container *ngFor="let quiz of quizData">
+  <mat-card *ngIf="quiz?.quizId === quizName">
+    <mat-card-header>
+      <div mat-card-avatar class="header-image"></div>
```

Comment:
Is there an actual image attached to this?

Reply:
changed it to img